src/liboslexec/batched_llvm_gen.cpp

```diff
@@ -4313,19 +4313,15 @@ LLVMGEN (llvm_gen_raytype)
     llvm::Value * sg = rop.sg_void_ptr();
     if (Name.is_constant()) {
         // We can statically determine the bit pattern
-        ustring name = ((ustring *)Name.data())[0];
-        llvm::Value *args[] = {
-            sg,
-            rop.ll.constant (rop.shadingsys().raytype_bit (name))};
-
-        llvm::Value *ret = rop.ll.call_function (rop.build_name("raytype_bit"), args);
```

Comment:
You should be able to remove
OSL_BATCHOP int __OSL_OP(raytype_bit)(void* bsg_, int bit)
from wide_shadingsys.cpp, and 
DECL(__OSL_OP(raytype_bit), "iXi")
from builtindecl_wide_xmacro.h